src/diffusers/models/resnet.py

```diff
@@ -373,59 +409,77 @@ def __init__(
         conv_shortcut=False,
         dropout=0.0,
         temb_channels=512,
-        groups=32,
+        groups: Optional[int] = 32,
```

Comment:
```suggestion
        groups=32,
```

Let's leave as is to not introduce too much difficult logic